src/ngAria/aria.js

```diff
@@ -377,6 +377,25 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
             });
           }
         }
+
+        if (elem[0].nodeName === 'A') {
+          if ($aria.config('bindRoleForClick') && !attr.href && !attr.xlinkHref && !attr.role) {
+              elem.attr('role', 'link');
+          }
+          if ($aria.config('bindKeypress')) {
+            elem.on('keypress', function(event) {
```

Comment:
Indeed switching to `keydown` seems more consistent with how native buttons work.
Yet, I suggest we make the change from `keypress` to `keydowm` in another PR (since it's used to several places) and possibly tests.

Also, does it mean that `bindKeypress` should be renames `bindKeydown` (which is a breaking change) ?
Leaving the name `bindKeypress` but binding to `keydown` isn't ideal either :smiley:
Maybe we could deprecate `bindKeypress` in favor of `bindKeydown` and remove it in `v1.6.x`...